app/components/elements/forms/select_field_component.rb

```diff
@@ -5,7 +5,7 @@ module Forms
     # Component for form select fields
     class SelectFieldComponent < ApplicationComponent
       def initialize(form:, field_name:, options:, required: false, hidden_label: false, # rubocop:disable Metrics/ParameterLists
-                     label: nil, help_text: nil, prompt: false, data: {})
+                     label: nil, help_text: nil, prompt: false, data: {}, label_classes: nil)
```

Comment:
The practice has been to use `[]` has the default value for a classes param instead of `nil`.

Reply:
Done.